geopyspark/rdd.py

```diff
@@ -57,10 +57,20 @@ def __init__(self, geopysc, rdd_type, srdd):
         self.rdd_type = rdd_type
         self.srdd = srdd
 
+    @property
     def layer_metadata(self):
         """Layer metadata associated with this layer"""
         return json.loads(self.srdd.layerMetadata())
 
+    @property
+    def zoom_level(self):
+        zoom = self.srdd.getZoom()
+
+        if zoom >= 0:
```

Comment:
I think its good idea to raise errors on python side as much as possible but in this case we can just return something that gets marshaled to python `None` with less surprise

```scala
  def getZoom: Integer = zoomLevel match {
    case Some(z) => new Integer(z)
    case None => null
  }
```